packages/checkout/widgets-lib/src/factory.ts

```diff
@@ -92,6 +94,15 @@ export class WidgetsFactory implements IWidgetsFactory {
           provider,
         }) as Widget<WidgetType.SALE> as Widget<T>;
       }
+      case WidgetType.CHECKOUT: {
+        if (this.sdk.config.environment === Environment.SANDBOX) {
```

Comment:
@jwhardwick can we please reverse the logic here to only throw if env is prod, because we have to allow dev deployments as well.